### PR TITLE
overwrite frm_page_container overflow style for translate page

### PR DESCRIPTION
**Pull request by AbdiTolesa**
Fixes https://github.com/Strategy11/formidable-pro/issues/4113

[issuse-4113.webm](https://user-images.githubusercontent.com/41271840/223715540-b289cdeb-1353-4c2a-82ea-19ba3fb7007e.webm)

### CODE REVIEW

File: css/frm_admin.css [hunk 1]
```diff
@@ -667,6 +667,10 @@ ul.frm_form_nav > li {
 	right: 0;
 }
 
+.frm-admin-page-translate .frm_wrap .frm_page_container {
+	overflow: scroll;
```
[the remark this answers is not in the document]
Comment (AbdiTolesa) [reply css/frm_admin.css]: Ok, just updated it accordingly @Crabcyborg.

**Comment by garretlaxton** [comment]
Scrolling works fine for me now. But the scrollbar only scrolls the WP admin menu.

**Comment by AbdiTolesa** [comment]
> Scrolling works fine for me now. But the scrollbar only scrolls the WP admin menu.

I noticed there were two scrollbars and you were trying the other one, that's why it didn't scroll the form for you, the inner scrollbar sort of scrolls but we don't want to have two scrollbars here at all so I have just fixed now. Thank you for catching that!

**Review by garretlaxton — APPROVED** [review]
Nice! This works great!

File: css/frm_admin.css [hunk 2]
```diff
@@ -667,6 +667,10 @@ ul.frm_form_nav > li {
 	right: 0;
 }
 
+.frm-admin-page-translate .frm_wrap .frm_page_container {
```
Comment (Crabcyborg) [inline css/frm_admin.css:670]: @AbdiTolesa It looks like this update breaks if I click save.

The action becomes `update_translations` on save so the page class becomes `frm-admin-page-update_translations` instead.

**Review by Crabcyborg — APPROVED** [review]
Thanks @AbdiTolesa!

**Pull request merged**